src/main/resources/application.properties

```diff
@@ -1,12 +1,2 @@
-data.file.path=src/main/resources/dataToFillMaps.json
-
-# jdbc.X
-jdbc.driverClassName=org.postgresql.Driver
-jdbc.url=jdbc:postgresql://localhost:5432/gym
-jdbc.user=armine
-jdbc.pass=0000
-
-# hibernate.X
-hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
-hibernate.show_sql=true
-hibernate.hbm2ddl.auto=create
\ No newline at end of file
+spring.profiles.active=dev
+spring.application.name=App
```

Comment:
As we talked it is better to move all repeatable properties to application.properties

Reply:
fixed